docs/getting-started/orchestration_quickstart.md

```diff
@@ -0,0 +1,162 @@
+# Orchestration Quickstart
+
+This guide is designed to get you to the point of successfully running a deployed Prefect flow in as few steps as possible. For a more comprehensive introduction to Prefect's core orchestration concepts, please follow our [tutorial pages](/tutorial/index/).
+
+### Step 1: [Install Prefect](/getting-started/installation/)
+
+We recommend installing Prefect using a Python virtual environment manager such as conda, or virtualenv.
+
+```bash 
+pip install -U prefect
+```
+### Step 2: Connect to Cloud or Self Host Prefect Server
```

Comment:
This may be an opportunity to use @discdiver's new tab feature for cloud or self-hosted :) For funzies